torch/_dynamo/variables/builder.py

```diff
@@ -562,6 +568,7 @@ def wrap_tensor(self, value: torch.Tensor):
                     guards=self.make_guards(GuardBuilder.TENSOR_MATCH),
                     should_specialize=self.tensor_should_specialize(),
                 )
+            graph_arg.fake_tensor = tensor_variable.proxy.node.meta['example_value']
```

Comment:
thanks i hate it lol

Reply:
this isn't... even guaranteed to be not None? Can you compute the proxy before you append to graphargs?